src/App.js

```diff
@@ -9,7 +10,7 @@ const article = {
 };
 
 const App = () => (
-  <h1>React Article</h1>
+  <Article title={article.title} text={article.text} date={article.date} />
```

Comment:
```suggestion
  <Article 
    title={article.title}
    text={article.text}
    date={article.date}
  />
```
Компоненты с нексолькими props записываются так

Reply:
done